frontend/src/metabase/components/Calendar.unit.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import moment from "moment-timezone";
+import { render, screen } from "__support__/ui";
+import MetabaseSettings from "metabase/lib/settings";
+import { updateMomentStartOfWeek } from "metabase/lib/i18n";
+import Calendar from "./Calendar";
+
+describe("Calendar", () => {
+  it("should render weekday short names", () => {
+    setup();
+
+    expect(
+      screen
+        .getAllByTestId("calendar-day-name")
+        .map((dayEl, index) => dayEl.textContent),
```

Comment:
nit: index not used

Reply:
shoudln't we have a lint rule or ts config settings for unused variables enabled?